src/py/flwr/server/server_test.py

```diff
@@ -88,24 +88,29 @@ def get_properties(
         self, ins: GetPropertiesIns, timeout: Optional[float]
     ) -> GetPropertiesRes:
         """Raise an Exception to simulate failure in the client."""
+        # pylint: disable-next=broad-exception-raised
         raise Exception()
```

Comment:
Should we raise `NotImplementedError` instead and remove the `# pylint: disable` comments?

Reply:
I used the `# pylint: disable` because the function docstrings specifically indicate that an `Exception` is raised, but I can update both if you prefer

Reply:
yes, let's update both to keep it consistent with the `SuccessClient` above (and also change the docstrings there)